constraints.pro

```diff
@@ -210,10 +210,12 @@ gen_enforced_field(WorkspaceCwd, 'repository.url', RepoUrl) :-
 % The license for all published packages must be MIT unless otherwise specified.
 gen_enforced_field(WorkspaceCwd, 'license', 'MIT') :-
   \+ workspace_field(WorkspaceCwd, 'private', true),
+  WorkspaceCwd \= 'packages/json-rpc-engine',
   WorkspaceCwd \= 'packages/eth-json-rpc-provider'.
 % The following published packages use an ISC license instead of MIT.
 gen_enforced_field(WorkspaceCwd, 'license', 'ISC') :-
   \+ workspace_field(WorkspaceCwd, 'private', true),
+  WorkspaceCwd == 'packages/json-rpc-engine',
   WorkspaceCwd == 'packages/eth-json-rpc-provider'.
```

Comment:
The `,` operator in Prolog means "and", so I'm not sure this would ever apply, as the name of a package cannot be two names at the same time. The `;` operator means "or", so I believe what we'd want is something like this:

``` suggestion
  (
    WorkspaceCwd == 'packages/json-rpc-engine' ;
    WorkspaceCwd == 'packages/eth-json-rpc-provider'
  ).

Reply:
Thanks for catching this!